integration-cli/docker_cli_run_unix_test.go

```diff
@@ -270,7 +270,8 @@ func (s *DockerSuite) TestRunOOMExitCode(c *check.C) {
 	errChan := make(chan error)
 	go func() {
 		defer close(errChan)
-		out, exitCode, _ := dockerCmdWithError("run", "-m", "4MB", "busybox", "sh", "-c", "x=a; while true; do x=$x$x$x$x; done")
+		//changes to 10MB from 4MB due to an issue that GCCGO fails to convert properly.
```

Comment:
@tiborvass The test requires higher memory limit with GCCGO, I cannot start the container with less than 10MB. The issue is being investigated. This is not a validation issue, I checked that validation passes fine when 4MB is specified. 
Another option is to skip this test for gccgo. 
I will correct the typo, if you are ok with the increase.